sdk/appconfiguration/app-configuration/sample-react/src/components/Page.tsx

```diff
@@ -4,14 +4,13 @@
   Licensed under the MIT license.
 */
 
-import React from "react";
```

Comment:
You were able to remove this and still compile and run? I thought the TypeScript compiler needed this to be present so it knew where to get the functions that the JSX gets compiled to. (Maybe it doesn't thought)

Reply:
I did test it. It was/is working fine. ⭐